src/context/AppProvider/hooks/useAccountsFetchQuery.tsx

```diff
@@ -115,7 +102,9 @@ export const useAccountsFetchQuery = () => {
       ).map(async accountIds => {
         const results = await Promise.allSettled(
           accountIds.map(async id => {
-            const result = await dispatch(getAccount.initiate({ accountId: id }))
+            const result = await dispatch(
+              getAccount.initiate({ accountId: id, upsertOnFetch: true }),
```

Comment:
`upsertOnFetch: true` added for referential stability between this call and the one below, to avoid overfetching